client/web/src/extensions/ExtensionCard.tsx

```diff
@@ -75,6 +77,18 @@ export const ExtensionCard = memo<Props>(function ExtensionCard({
         [extension]
     )
 
+    const actionableErrorMessage = (error: Error) => {
+        let errorMessage;
+
+        if (error.message.startsWith('invalid settings') && settingsURL) {
+            errorMessage = <>Could not enable / disable {name}. Edit your <a href={settingsURL}>user settings</a> to fix this error.</>
```

Comment:
```suggestion
            errorMessage = <>Could not enable / disable {name} due to an invalid setting ({put_setting_line_here}). Edit your <a href={settingsURL}>user settings</a> to fix this error.</>
```

This is much better! I'll say we can get away with doing "enable / disable" even though it's not the prettiest because this is an uncommon path, and the more important action is giving them a way to unblock themselves. I do wonder if we can print just a little more context so they know what to do when they get to settings – something like this?

Reply:
Ah, sorry if I wasn't clear -- I was going off your original screenshot in this PR and noticed "Invalid settings: search.scopes.0..." I'd also be happy to just print that whole message in parentheses here -- our users our primarily and largely developers who won't be shocked by a non-conversational phrase included in this. Would that be fast and still give the user the additional information?

Reply:
Oh yeah makes sense!! That's a good idea